content/03-front-end-libraries/react/add-comments-in-jsx.md

```diff
@@ -1,42 +1,42 @@
 ---
 id: 5a24bbe0dba28a8d3cbd4c5e
-title: Add Comments in JSX
+title: Adicionando comentários em JSX
 challengeType: 6
 forumTopicId: 301376
 dashedName: add-comments-in-jsx
 ---
 
 # --description--
 
-JSX is a syntax that gets compiled into valid JavaScript. Sometimes, for readability, you might need to add comments to your code. Like most programming languages, JSX has its own way to do this.
+JSX é uma sintáxe que é compilada para JavaScript válido. por vezes, por questões de leitura, você precise adicionar comentários no seu código. Assim como todas linguagens de programação, JSX tem o seu próprio jeito de fazer isso.
```

Comment:
'Sintaxe' doesn't have accent mark.

This sentence starts with capital letter:
> . por vezes, [...]

"for readability" might be translated using something like "para facilitar a leitura" or "para tornar a leitura fácil". It's a suggestion... 😊

Please, review the meaning of the verb might. I suggest [this post about it]( https://www.ingles200h.com/gramatica-em-ingles/ ).